tests-dynamic.scm

```diff
@@ -164,15 +165,50 @@
         '(if (null? xs) ys
              (cons (car xs)
                    (append (cdr xs) ys))))))
+(test
+ (run* (q) (fresh (x y)
+                 (== q (list x y))
+                 (dynamic x)
+                 (== x y)
+                 (== y 5)
+                 (== y 6)))
+ '())
+
+(test 
+ (run* (q) (fresh (x y)
+                 (== q (list x y))
+                 (dynamic x)
+                 (== y 5)
+                 (== x y)
+                 (== y 6)))
+ '())
+
+(test
+ (run* (q) (fresh (x y)
+                 (== q (list x y))
+                 (dynamic x)
+                 (== y 5)
+                 (== y 6)
+                 (== x y)))
```

Comment:
can you also add the test where the == on x y is before the dynamic call

Reply:
Done